src/main/java/duke/parser/Parser.java

```diff
@@ -69,7 +69,7 @@ public static Command parse(String input) {
      */
     private static String getArguments(String input) throws ArrayIndexOutOfBoundsException {
         try {
-            return input.split(" ", INPUT_STRING_SPLIT_LIMIT)[ARGUMENTS_INDEX];
+            return input.split(" ", INPUT_STRING_SPLIT_LENGTH_LIMIT)[ARGUMENTS_INDEX];
```

Comment:
It might work with just throwing the exception e without catching it then throwing in the catch block?

Reply:
Thanks for the feedback; I didn't know about that. I'll look into it!